### ISSUE TITLE
Not using GPU

### ISSUE THREAD

**Issue by diegomora02** [Anjok07/ultimatevocalremovergui#880]
![image](https://github.com/Anjok07/ultimatevocalremovergui/assets/31973242/480e0eb5-3f10-457c-9d05-7091a6a3a39a) 
![image](https://github.com/Anjok07/ultimatevocalremovergui/assets/31973242/06b45769-c582-452f-9b84-6448f603e2ee)

I have a laptop with an Nvidia GTX 1650 and it is not using it despite having GPU processing enabled, I have downloaded the most recent drivers but it is clearly using CPU

**Comment by AngusGhoul** [comment]
I have the same fuckin problem and it´s really annoying, did you find the solution?

**Comment by Anjok07** [comment]
@AngusGhoul I made some important changes the latest release that should have resolved this issue for most people. What type of GPU do you have?

@diegomora02 A majority of models in UVR require at least 8GB's of V-RAM (6 being the absolute bare minimum) and it looks like you only have 4. In some cases if the application can't run on the GPU, it defaults to the CPU instead.

**Comment by AngusGhoul** [comment]
@Anjok07 yes I start to have this problem after I updated yesterday. I have a Geforce 3050rtx. but it´s weird because works sometimes, and sometimes not.

**Comment by AngusGhoul** [comment]
this one is my version Ultimate Vocal Remover v5.6.0 [2023-10-18 13:58:59]

**Comment by Anjok07** [comment]
@AngusGhoul Try these 2 things 

1. If you're using MDX-Net, choose any segment size other than 256
2. Choose your GPU directly (see image). If you don't see a GPU selection that doesn't look like the image below, please reinstall the patch [here](https://github.com/Anjok07/ultimatevocalremovergui/releases/download/v5.6/UVR_Patch_10_6_23_4_27.exe)
![choose_gpu](https://github.com/Anjok07/ultimatevocalremovergui/assets/68268275/81d649c2-8cea-4b00-abf4-545834ca3e07)

**Comment by AngusGhoul** [comment]
Yes, I selected the gpu manually before. I will send you screenshots of my task manager. Like I said, the program was working before the last patch. I dont know what to do but was working before :([
![imagen_2023-10-20_205203096](https://github.com/Anjok07/ultimatevocalremovergui/assets/99844524/b787190a-ee06-467a-ba86-52ab9760b946)
![imagen_2023-10-20_205237900](https://github.com/Anjok07/ultimatevocalremovergui/assets/99844524/fc986e23-344b-4358-b268-0b07d0f1cd42)
![imagen_2023-10-20_205300449](https://github.com/Anjok07/ultimatevocalremovergui/assets/99844524/e097831d-2dee-41be-89c1-e92ffab1cfab)
](url)

**Comment by ChrisTG742** [comment]
I think a 1030 has a too old revision of CUDA. But even if it would work, the CUDA-cores do not have enough power to bring any performance boost compared to a decent CPU. I believe everything below GT1080 will not make sense.

**Comment by luciano-work** [comment]
@Anjok07 I have the same problem using Windows Manual Installation,
Is there a way to apply the last patch on manual installation?

**Comment by Jomaexp** [comment]
I have rtx 5080 and when I try the GPU conversion the program doesn`t do anything. I have to end the process. With CPU conversion al works well. Any idea? Thanks.

**Comment by jackspends-hash** [comment]
Similar issue with RTX 5060-Ti. I see the GPU usage spike for a second, the process starts, and immediately gets to 5% or 10% and then freezes. I let it run normally and it took 27 minutes to finish the process.

This happens with Demucs + MDX. I wasn't having this issue with my 3070-Ti, but I recently swapped cards and this started happening after a fresh install.

**Comment by senduora** [comment]
> Similar issue with RTX 5060-Ti. I see the GPU usage spike for a second, the process starts, and immediately gets to 5% or 10% and then freezes. I let it run normally and it took 27 minutes to finish the process.
> 
> This happens with Demucs + MDX. I wasn't having this issue with my 3070-Ti, but I recently swapped cards and this started happening after a fresh install.

same, upgraded from 3080 to 5070 and have this problem

**Comment by jackspends-hash** [comment]
found a solution - a patch was released by Anjok for 5000-series cards + Windows. this link will download a patch that you install directly on top of your current UVR.

[Download](https://www.mediafire.com/file_premium/4jg10r9wa3tujav/UVR_Patch_4_24_25_20_11_BETA_full_cuda_12.8.zip/file)

**Comment by tiger3645** [comment]
Thanks @jackspends-hash @Anjok07! The patch uploaded [here](https://github.com/Anjok07/ultimatevocalremovergui/issues/880#issuecomment-3981834001) worked perfectly and now inference is running correctly on my RTX 5000 series card.

**Comment by ChrisTG742** [comment]
Too bad that this project seems to be abandoned since a while. There are dozen of great ideas and pull-requests open.

**Comment by BanksRao** [comment]
@jackspends-hash thx. Same issue. Solves 5070Ti
